src/common/pool_map.c

```diff
@@ -934,12 +934,10 @@ pool_map_finalise(struct pool_map *map)
  * Install a component tree to a pool map.
  *
  * \param map		[IN]	The pool map to be initialized.
- * \param activate	[IN]	Activate pool components.
```

Comment:
This "activate" logic caused a huge number of problems for addition, as many different components would get the pool map update with the new ranks in the NEW state, then helpfully "activate" them, causing pool version desync. All of the addition testing was with this activation code removed - it doesn't seem to be important to being able to run the servers, as apparently components are set to UPIN elsewhere during startup.